### PR TITLE
refactor: move event handlers to constructors

### PR DESCRIPTION
**Pull request by kyubisation**
This is the recommended approach for event handlers for web components.

### CODE REVIEW

File: tools/generate-component/boilerplate/component.ts [hunk 1]
```diff
@@ -36,13 +34,6 @@ export class __nameUpperCase__ extends LitElement {
     this._myEvent.emit();
   }
 
-  public override connectedCallback(): void {
-    super.connectedCallback();
-    const signal = this._abort.signal;
-    this.addEventListener('click', () => this._onClickFn(), { signal });
```
Comment (jeripeierSBB) [inline tools/generate-component/boilerplate/component.ts:42]: I suggest to add the click event listener also here to the constructor. We still have the _onClickFn() function which is unused now.

**Review by jeripeierSBB — APPROVED** [review]
LGTM, thanks!

File: tools/generate-component/boilerplate/component.ts (hunk 1 above)
Comment (kyubisation) [reply tools/generate-component/boilerplate/component.ts:42]: I think I would rather remove it from the template entirely (and maybe more). We have enough examples which can be used as reference.
What do you think?

Comment (jeripeierSBB) [reply tools/generate-component/boilerplate/component.ts:42]: ok as well. boilerplate usage is rare anyways :)

File: tools/generate-component/boilerplate/component.ts [hunk 2]
```diff
@@ -16,40 +14,12 @@ import style from './__noPrefixName__.scss?lit&inline';
 export class __nameUpperCase__ extends LitElement {
```
Comment (jeripeierSBB) [inline tools/generate-component/boilerplate/component.ts:14]: jsdoc of myEventName should be removed to be consistent

**Pull request merged**